### PR TITLE
add Palora.vercel.app

### PR DESCRIPTION
**Pull request by arviinmo**
This PR is:

- [x] Adding a new domain
- [ ] Updating existing domain **size**
- [ ] Changing domain name
- [ ] Removing existing domain from list
- [ ] Website code changes (512kb.club site)
- [ ] Other not listed

- [x] I used the uncompressed size of the site
- [x] I have included a link to the GTMetrix report
- [ ] The domain is in the correct alphabetical order
- [x] This site is not a ultra lightweight site
- [x] The following information is filled identical to the data file

***I confirm that I have read the [FAQ section](https://512kb.club/faq), particularly the two red items around minimal pages and inappropriate content and I attest that this site is neither of these things.***

- [ ] Check to confirm

```
- domain:
  url:
  size:
  last_checked:
```

GTMetrix Report: https://gtmetrix.com/reports/palora.vercel.app/qpTsAkfC/

### CODE REVIEW

File: _data/sites.yml [hunk 1]
```diff
@@ -2407,3 +2407,8 @@
   url: https://zserge.com/
   size: 20.6
   last_checked: 2022-05-13
+
+- domain: palora.vercel.app
+  url: https://palora.vercel.app/
+  size: 403
+  last_checked: 2021-7-11
```
Comment (garritfra) [inline _data/sites.yml:2414]: You probably meant to use the current year here.🙂

Comment (arviinmo) [reply _data/sites.yml:2414]: but you can edit it for me!

**Review by garritfra — CHANGES_REQUESTED** [review]
Missed this during the review, but the domains should be ordered alphabetically.

**Comment by arviinmo** [comment]
> Missed this during the review, but the domains should be ordered alphabetically.

done!

**Comment by garritfra** [comment]
LGTM!

**Review by garritfra — APPROVED** [review]

**Pull request merged**